servicetalk-grpc-internal/build.gradle

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright © 2019, 2021 Apple Inc. and the ServiceTalk project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+apply plugin: "io.servicetalk.servicetalk-gradle-plugin-internal-library"
+
+dependencies {
+  api project(":servicetalk-annotations")
```

Comment:
`servicetalk-annotations` is always `implementation`

Reply:
OK. It seems like it could be `api` if anybody wanted to access the annotations at runtime, but I will make the change.

Reply:
`implementation` gives users the dependency at runtime, `api` gives it at compile time